pages/common/ftp.md

```diff
@@ -0,0 +1,27 @@
+# ftp
+
+> Internet File Transfer Protocol
+
+- Connect to a FTP server:
+
+`ftp {{ftp.example.com}}`
+
+- Switch to binary transfer mode (graphics, compressed files etc):
+
+`binary`
+
+- Turn off interactive prompting for multiple file transfers:
+
+`prompt off`
+
+- Download multiple files (glob expression):
+
+`mget {{*.png}}`
+
+- Upload multiple files (glob expression):
+
+`mput {{*.zip}}`
+
+- Delete multiple files on the remote machine (glob expression):
+
+`mdelete {{*.txt}}`
```

Comment:
We could not mention the single-file ones; the unintuitivess comes from the tool itself, so there isn't much we could do: either we'd recommend single-file commands that are not really useful, or we'd mention commands that contain an otherwise unexplained prefix for multiple. Let's just use the m versions, I suppose it's the lesser of the two evils.